### PR TITLE
Feature/nis 1522

### PR DESCRIPTION
**Pull request by Leblyad**

### CODE REVIEW

File: InnoClinic.ServicesAPI/Presentation/Controllers/ServicesController.cs [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
 using ServicesAPI.Core.Entities.DataTransferObject;
 using ServicesAPI.Core.Entities.QueryParameters;
 using ServicesAPI.Core.Services.Abstractions;
 
 namespace ServicesAPI.Controllers
 {
-    [Route("api/service")]
+    [Route("api/services")]
```
Comment (kseniya-kalita) [inline InnoClinic.ServicesAPI/Presentation/Controllers/ServicesController.cs:9]: you can just use [controller]

File: InnoClinic.ServicesAPI/Presentation/Controllers/ServicesController.cs [hunk 2]
```diff
@@ -16,6 +18,7 @@ public ServicesController(IServiceManager serviceManager)
             _serviceManager = serviceManager;
         }
 
+        [Authorize(Roles = "Doctor")]
```
Comment (kseniya-kalita) [inline InnoClinic.ServicesAPI/Presentation/Controllers/ServicesController.cs:21]: Don't hardcode strings

File: InnoClinic.ServicesAPI/Presentation/Extensions/ServiceExtensions.cs [hunk 3]
```diff
@@ -55,5 +57,47 @@ public static void ConfigureValidatorsAndControllers(this IServiceCollection ser
                 .AddFluentValidation(c =>
                 c.RegisterValidatorsFromAssembly(Assembly.GetExecutingAssembly()));
         }
+
+        public static void ConfigureJWTAuthentification(this IServiceCollection services)
+        {
+            services.AddAuthentication("Bearer")
+                    .AddJwtBearer("Bearer", options =>
+                    {
+                        options.Authority = "https://localhost:7141";
```
Comment (kseniya-kalita) [inline InnoClinic.ServicesAPI/Presentation/Extensions/ServiceExtensions.cs:66]: hardcoded string, please move it

File: InnoClinic.ServicesAPI/Presentation/Extensions/ServiceExtensions.cs [hunk 4]
```diff
@@ -55,5 +57,47 @@ public static void ConfigureValidatorsAndControllers(this IServiceCollection ser
...
+                        options.Audience = "APIClient";
+                        options.TokenValidationParameters = new TokenValidationParameters
+                        {
+                            ValidateAudience = false
```
Comment (kseniya-kalita) [inline InnoClinic.ServicesAPI/Presentation/Extensions/ServiceExtensions.cs:70]: please change to true due to the security issues

File: InnoClinic.ServicesAPI/Presentation/Extensions/ServiceExtensions.cs [hunk 5]
```diff
@@ -55,5 +57,47 @@ public static void ConfigureValidatorsAndControllers(this IServiceCollection ser
                 .AddFluentValidation(c =>
```
Comment (kseniya-kalita) [inline InnoClinic.ServicesAPI/Presentation/Extensions/ServiceExtensions.cs:57]: Don't hardcode Assembly (upper) now, it's impossible to run update-database now

File: InnoClinic.ServicesAPI/Presentation/Controllers/ServicesController.cs (hunk 1 above)
Comment (Leblyad) [reply InnoClinic.ServicesAPI/Presentation/Controllers/ServicesController.cs:9]: It will written with upper case.

**Pull request merged**